common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -228,7 +233,7 @@ def student_view(self, context):
                 'type': child.get_icon_class(),
                 'id': child.scope_ids.usage_id.to_deprecated_string(),
                 'bookmarked': is_bookmarked,
-                'path': " > ".join(display_names + [child.display_name or '']),
+                'path': " > ".join(display_names + [markupsafe.escape(child.display_name_with_default)]),
```

Comment:
@cahrens  display_name_with_default_escaped is deprecated that's why i didn't use it.